src/test/scala/group/scala/karazin/circe/literal/extras/PlainPrimitiveEncodeSuite.scala

```diff
@@ -1297,3 +1298,45 @@ class PlainPrimitiveEncodeSuite extends munit.ScalaCheckSuite:
       assertEquals(result, expected)
     }
   }
+
+  property("inlined Currency value") {
+
+    case class Primitive(value: Currency) derives Codec.AsObject
+
+    extension (inline sc: StringContext)
+      inline def encode(inline args: Any*): Json =
+        $ {macros.encode[Currency]('sc, 'args)}
+
+    forAll { (currency: Currency) =>
+
+      val primitive = Primitive(currency)
+
+      val result: Json = currency.asJson
+
+      val expected: Json = encode"${primitive.value}"
```

Comment:
Due to the issue above, should be `"value": ${primitive.value}`

Reply:
Done